src/contents/articles/hello-world/index.md

```diff
@@ -27,4 +25,4 @@ We want to grow as software developers but we also want to grow as a people. As
 
```

Comment:
I think the pillars could be mentioned as we're trying to stress that we're a club that's far more than tech event organisation. It should be important to highlight how holistic our goals and activities are. I agree that the some information could be provided on the pillars, or a sample of our planned activities to illustrate how we uphold them.